src/TraitsV2/Trait.class.st

```diff
@@ -268,6 +268,17 @@ Trait >> isClassTrait [
 	^ false
 ]
 
+{ #category : #'as yet unclassified' }
+Trait >> isEmpty [ 
+	"Since we can have potential an empty array in the fluid definition (to be revisited) 
+	when the trait composition is aTrait it received the message isEmpty and would break
+	traitComposition 
+		ifNotNil: [ :t | t isEmpty ifFalse: [shiftClassBuilder traitComposition: traitComposition ]]."
+
+	^ false
```

Comment:
And here.

Reply:
Like what? runtime checks? 
Right now I do not have the answer. First I want to know what is the argument of uses: because it can be a trait or a trait composition and I would prefer to have traits: with only one collection, now the problem is that T1 + T2 is a trait so it means that traits: argument would be collection with only one argument so it sucks.  
So this is a problem because there is no intersection between traits: and uses: arguments.
I do not like uses: because it looks strange. I would prefer trait: because at the end if we keep T1 + T2 then I get a single trait (composed) in any way I turn it around.
So may be the solution is to have trait: and remove uses: and in that case I will be able to remove isEmpty. 
But we should decide what do we want for the fluid syntax for trait.